### ISSUE TITLE
Config FAB_INDEX_VIEW  doesn't work

### ISSUE THREAD

**Issue by Mimameid** [dpgaspar/Flask-AppBuilder#1879]
### Environment

Flask-Appbuilder version: 4.1.2

### Describe the expected results

When the configuration option `FAB_INDEX_VIEW` is defined, the application should load given IndexView.

### Describe the actual results

I get the following error:

```
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ibrahimk/projects/incubation/seidr/a/app/__init__.py", line 16, in <module>
    appbuilder = AppBuilder(app, db.session)
  File "/home/ibrahimk/projects/incubation/seidr/venv/lib/python3.9/site-packages/flask_appbuilder/base.py", line 147, in __init__
    self.init_app(app, session)
  File "/home/ibrahimk/projects/incubation/seidr/venv/lib/python3.9/site-packages/flask_appbuilder/base.py", line 177, in init_app
    if isinstance(view, BaseView):
NameError: name 'BaseView' is not defined

Process finished with exit code 1

```

**Comment by phildum** [comment]
Seeing the same when trying to upgrade Superset to 2.0.0rc1. We have FAB_INDEX_VIEW set in config.py/values.yaml

```
Traceback (most recent call last):
  File "/app/superset/app.py", line 37, in create_app
    app_initializer.init_app()
  File "/app/superset/initialization/__init__.py", line 460, in init_app
    self.init_app_in_ctx()
  File "/app/superset/initialization/__init__.py", line 408, in init_app_in_ctx
    self.configure_fab()
  File "/app/superset/initialization/__init__.py", line 500, in configure_fab
    appbuilder.init_app(self.superset_app, db.session)
  File "/usr/local/lib/python3.8/site-packages/flask_appbuilder/base.py", line 177, in init_app
    if isinstance(view, BaseView):
NameError: name 'BaseView' is not defined
```

**Comment by dpgaspar** [comment]
fix released on 4.1.3
